cwms-http-client/src/main/java/mil/army/usace/hec/cwms/http/client/CwmsHttpLoggingInterceptorLogger.java

```diff
@@ -0,0 +1,14 @@
+package mil.army.usace.hec.cwms.http.client;
+
+import okhttp3.logging.HttpLoggingInterceptor;
+
+final class CwmsHttpLoggingInterceptorLogger implements HttpLoggingInterceptor.Logger {
+
+    @Override
+    public void log(String message) {
+        HttpLoggingInterceptor.Logger logger = HttpLoggingInterceptor.Logger.DEFAULT;
+        if (!message.startsWith("Authorization:")) {
```

Comment:
This is why I was worried about logging more details, we have to be very careful about what we allow in here. This should be case insensitive. I also think there are some common headers that we can add in here for future proofing.

Reply:
Cookie is the other one listed by OkHttp documentation

Reply:
I updated to override the intercept to check for POST requests, and log the bodies. This way we don't have to worry about which headers can/can't be exposed.

Reply:
The class instance that put "Authorization:" into the message should provide the filter business that prevents "Authorization:" from being logged.

Reply:
I updated this on my latest push to remove this log class altogether, and replaced it with an interceptor that logs the POST body, but never the headers. My thought here is that we should never be passing in passwords, etc. in the post body anyways, so we can log the post body, but not the headers.

Reply:
I believe the "Authorization:" just comes from the request headers. To achieve filtering like that, will have to write our own custom Logging interceptor.

Reply:
Updated to make use of redactHeader method, and have the Authenticator handle the redaction.